### ISSUE TITLE
cnf header for example in README.markdown

### ISSUE THREAD

**Issue by ajohnson1** [msoos/cryptominisat#474]
A very minor issue with the readme is where it says:
```
p cnf 2 3
1 0
-2 0
-1 2 3 0
```
>The files has 3 clauses and 2 variables, this is reflected in the header `p cnf 2 3`.

The example actually has 3 variables, so perhaps it should read:
The file has 3 variables and 3 clauses, this is reflected in the header `p cnf 3 3`.

(It's then not so clear which '3' is which so reverse the order of 'variables' and 'clauses' to match the order in the header.)

Similarly
```
p cnf 2 4
1 0
-2 0
-3 0
-1 2 3 0
```
should have `p cnf 3 4`. You could add a comment

The file has 3 variables and 4 clauses, this is reflected in the header `p cnf 3 4`.

**Comment by msoos** [comment]
Please submit a pull request to fix this. Thanks.

**Comment by ajohnson1** [comment]
Fixed by pull #475

**Comment by msoos** [comment]
Thanks!
